core/reactor.h

```diff
@@ -590,6 +614,37 @@ void print_snapshot(void);
  */
 void request_stop(void);
 
+/**
+ * Allocate memory using calloc (so the allocated memory is zeroed out)
```

Comment:
I think in the interest of generality, this could read "Allocate zeroed out memory", especially if it's in platform.h.